uber/site_sections/accounts.py

```diff
@@ -88,7 +88,7 @@ def delete(self, session, id, **params):
         session.delete(session.admin_account(id))
         raise HTTPRedirect('index?message={}', 'Account deleted')
 
-    @site_mappable
+    @site_mappable(download=True)
```

Comment:
This isn't a downloadable file! It lets you assign admin accounts to volunteers in bulk.

Reply:
Oops, should be fixed!